modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/utils/loadIssues.js

```diff
@@ -37,11 +37,11 @@ export default function loadIssues({
 					type: SET_ISSUES,
 				});
 			})
-			.catch(({googlePageSpeedError = {}}) => {
-				const {error} = googlePageSpeedError;
+			.catch((error = {}) => {
+				const {googlePageSpeedError: {error: gpsError} = {}} = error;
```

Comment:
I think this is a little bit difficult to read, could it look better with an `if`? What do you think? 😄

Reply:
I'm also not used to this kind of nested deconstructing because it is difficult to read for me, but that is just my preference.